From 67f696966fb39a8a2e2feeae53a0d7446cfb5c33 Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Mon, 28 Apr 2008 14:47:11 +0800
Subject: [PATCH] ENGR00074515 A warning dbg msg should be removed for eSDHC

One dbg msg shold be removed in the release version.

Signed-off-by: Richard Zhu <r65037@freescale.com>
---
 drivers/mmc/host/mx_sdhci.c |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/drivers/mmc/host/mx_sdhci.c b/drivers/mmc/host/mx_sdhci.c
index 05b16bb..78baeef 100644
--- a/drivers/mmc/host/mx_sdhci.c
+++ b/drivers/mmc/host/mx_sdhci.c
@@ -439,11 +439,8 @@ static void sdhci_prepare_data(struct sdhci_host *host, struct mmc_data *data)
 			break;
 	}
 
-	if (count >= 0xF) {
-		printk(KERN_WARNING "%s: Too large timeout requested!\n",
-		       mmc_hostname(host->mmc));
+	if (count >= 0xF)
 		count = 0xE;
-	}
 
 	count =
 	    (count << 16) | (readl(host->ioaddr + SDHCI_CLOCK_CONTROL) &
-- 
1.5.4.4

